### PR TITLE
Modify components folder path in documentation

### PR DESCRIPTION
**Pull request by vinayada1**
Thank you for helping make the Dapr documentation better!

If you are a new contributor, please see the [this contribution guidance](https://github.com/dapr/docs/blob/master/contributing/README.md) which helps keep the Dapr documentation readable, consistent and useful for Dapr users.

If you are contributing a new authored "How To" article, [this template](https://github.com/dapr/docs/blob/master/contributing/howto-template.md) was created to assist you.

In addition, please fill out the following to help reviewers understand this pull request:

## Description

default components path has been modified. Modify documentation accordingly

## Issue reference

_Please reference the issue this PR will close: #620

### CODE REVIEW

File: walkthroughs/darprun.md [hunk 1]
```diff
@@ -7,15 +7,15 @@ Terminology used below:
 - Dapr CLI - the Dapr command line tool.  The binary name is dapr (dapr.exe on Windows)
 - Dapr runtime - this runs alongside each app.  The binary name is daprd (daprd.exe on Windows)
 
-In self hosting mode, running `dapr init` copies the Dapr runtime onto your box and starts the placement service (used for actors) and Redis in containers.  These must be present before running `dapr run`.
+In self hosting mode, running `dapr init` copies the Dapr runtime onto your box and starts the placement service (used for actors) and Redis in containers.  These must be present before running `dapr run`. The Dapr CLI also creates the default components directory under `~\.dapr\components` if it does not already exist.
```
Comment (amanbha) [inline walkthroughs/darprun.md:10]: Clarify that that path is for Linux/MacOS, also add path for Windows.

File: walkthroughs/darprun.md [hunk 2]
```diff
@@ -7,15 +7,15 @@ Terminology used below:
...
 
 What happens when `dapr run` is executed?  
 
 ```bash
 dapr run --app-id nodeapp --app-port 3000 --port 3500 node app.js
 ```
 
-First, the Dapr CLI creates the `\components` directory if it does not not already exist, and writes two component files representing the default state store and the default message bus: `redis.yaml` and `redis_messagebus.yaml`, respectively.  [Code](https://github.com/dapr/cli/blob/d585612185a4a525c05fb62b86e288ccad510006/pkg/standalone/run.go#L254-L288).
+First, the Dapr CLI uses the default components directory under `~\.dapr\components` to write two component files representing the default state store and the default message bus: `redis.yaml` and `redis_messagebus.yaml`, respectively.  [Code](https://github.com/dapr/cli/blob/d585612185a4a525c05fb62b86e288ccad510006/pkg/standalone/run.go#L254-L288).
```
Comment (LMWF) [inline walkthroughs/darprun.md:18]: Today those 2 yamls are named "statestore.yaml" and "pubsub.yaml" respectively.

Comment (LMWF) [reply walkthroughs/darprun.md:18]: also can you rename "default message bus" should be changed to "default pub/sub".

Comment (pruthvidhodda) [inline walkthroughs/darprun.md:18]: This one needs to be updated, it should be pubsub.

Comment (pruthvidhodda) [reply walkthroughs/darprun.md:18]: I see @LMWF already mentioned this above, ignore this.

Comment (pruthvidhodda) [reply walkthroughs/darprun.md:18]: Please update the link to the code.

File: walkthroughs/darprun.md (hunk 1 above)
Comment (vinayada1) [reply walkthroughs/darprun.md:10]: fixed

File: walkthroughs/darprun.md (hunk 2 above)
Comment (vinayada1) [reply walkthroughs/darprun.md:18]: fixed

Comment (vinayada1) [reply walkthroughs/darprun.md:18]: done

**Comment by vinayada1** [comment]
> Can we rename this file to daprrun.md?

Done

**Pull request merged**